### ISSUE TITLE
Sonarr and Radarr Problem

### ISSUE THREAD

**Issue by Lanneer487** [MichaIng/DietPi#1566]
When I want to install sonarr and radarr on the latest version of dietpi, The installation completed successfully. The problem is that they won't boot and can't get started manually.

**Comment by symbios24** [comment]
Hi, di you access the web interface of sonarr for example from another computer connected to the same network thought your internet browser url = http://192.168.0.100:8989 ??

**Comment by Lanneer487** [comment]
Yes, but before I updated to the latest version everything worked. So I did a clean install of dietpi and installed sonarr but it wont start. Same problem for radarr. when I look at the service status, I get this as feedback:
● sonarr.service - Sonarr (NzbDrone) Daemon
   Loaded: loaded (/etc/systemd/system/sonarr.service; disabled; vendor preset: 
enabled)
   Active: activating (auto-restart) (Result: exit-code) since Wed 2018-02-28 15
:07:54 GMT; 2s ago
  Process: 3769 ExecStart=/usr/bin/mono --debug /opt/NzbDrone/NzbDrone.exe -nobr
owser (code=exited, status=1/FAILURE)
 Main PID: 3769 (code=exited, status=1/FAILURE)

Feb 28 15:07:54 RaspberryPi systemd[1]: sonarr.service: Main process exi
ted, code=exited, status=1/FAILURE
Feb 28 15:07:54 RaspberryPi systemd[1]: sonarr.service: Unit entered fai
led state.
Feb 28 15:07:54 RaspberryPi systemd[1]: sonarr.service: Failed with resu
lt 'exit-code'.

**Comment by Fourdee** [comment]
🈴 RPi 3 test ```dietpi-software install 144 145``` :
- 🈴 Mono Fails install due to ```/tmp tmpfs``` being full. Filled with mono.
```G_AGI: mono-devel mediainfo sqlite3 libmono-cil-dev```
- Ref: https://github.com/Fourdee/DietPi/issues/1027
- ```umount: /tmp: target is busy```
- Mono uses ```502M``` during APT install.

Can't see any options to change only dpkg/apt unpack directory. We may need to remove ```/tmp``` from 1GB< systems.
- Interesting, we can set a higher ```/tmp tmpfs``` size than available RAM. Should use SWAP in that instance.
```
root@DietPi:~# df -h
Filesystem      Size  Used Avail Use% Mounted on
tmpfs           2.0G     0  2.0G   0% /tmp
```

---

```
Executing: /tmp/apt-key-gpghome.steDtXb54V/gpg.1.sh --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
gpg: failed to start the dirmngr '/usr/bin/dirmngr': No such file or directory
gpg: connecting dirmngr at '/tmp/apt-key-gpghome.steDtXb54V/S.dirmngr' failed: No such file or directory
gpg: keyserver receive failed: No dirmngr
```

- 🈯️ (https://github.com/Fourdee/DietPi/issues/1388#issuecomment-369327576) make ```dirmngr``` standard pre-req DietPi package for all systems.

---

🈴 Service auto restarting confirmed
- Running both binaries at same time manually is fine, although ```Press enter to exit...``` most likely a end prompt to mono binary finishing:
```
/usr/bin/mono --debug /opt/Radarr/Radarr.exe -nobrowser
/usr/bin/mono --debug /opt/NzbDrone/NzbDrone.exe -nobrowser
```

- Issue persists using Raspbian pool for mono install ```deb http://download.mono-project.com/repo/debian raspbianstretch main```

- 🈯️ Sonarr resolved by using ```develop``` repo branch
- (https://github.com/Radarr/Radarr/issues/2585#issue-300794900) Radarr, already using dev:
```
root@DietPi:~# curl -s "$INSTALL_URL_ADDRESS" | grep 'linux.tar.gz' | grep 'browser_download_url' | head -1
        "browser_download_url": "https://github.com/Radarr/Radarr/releases/download/v0.2.0.980/Radarr.develop.0.2.0.980.linux.tar.gz"
```

**Comment by Lanneer487** [comment]
<img width="870" alt="schermafbeelding 2018-02-28 om 17 42 07" src="https://user-images.githubusercontent.com/9874656/36800003-c2ad3226-1cae-11e8-85af-db2b7f1a58f3.png">
How can I fix this? it keeps restarting

**Comment by Fourdee** [comment]
Ok, minus our issue with ```/tmp``` out of space (https://github.com/Fourdee/DietPi/issues/1566#issuecomment-369296662)

This is an issue with Mono version used (```5.10``` latest via http://download.mono-project.com/repo/debian):
- Radarr is pending update to work with latest version of Mono: https://github.com/Radarr/Radarr/issues/2585#issue-300794900
- Sonarr fix is already done, confirmed using develop repo install.

Regardless of above, i'll set Sonarr to develop repo install by default (in line with Radarr).

**Comment by symbios24** [comment]
Hello i installed also both radarr and sonarr in latest DietPi stretch in a raspberryPi 3
Sonarr web interface loads fine but Radarr web interface is not loading can this be fixed manual?

Thanks

**Comment by symbios24** [comment]
Hello i installed also both radarr and sonarr in latest DietPi stretch in a raspberryPi 3
Sonarr web interface loads fine but Radarr web interface is not loading and as i see on boot the Radarr service is running, can this be fixed manual?

Thanks

**Comment by Lanneer487** [comment]
I used this link to fix my problem and Radarr is working again.
[https://forums.sonarr.tv/t/mono-5-10-update-broke-sonarr/17800/19](https://forums.sonarr.tv/t/mono-5-10-update-broke-sonarr/17800/19)

**Comment by symbios24** [comment]
Hi, But the fix is for Sonarr not Radarr 
Will this fix Radarr?

**Comment by Lanneer487** [comment]
yes just change /opt/NzbDrone/ to /opt/Radarr/

**Comment by samsepiol59** [comment]
@roelfoubert I did (Sonarr is working, Radarr is not) but nothing happened, I'm still unable to access radarr at :7878!

I copied the file in /opt/Radarr, chmod'ed it and restarted the service - but nothing... am I missing something?

EDIT: now it seems that Jackett is down too??

**Comment by symbios24** [comment]
samsepiol59 you installed both on latest DietPi?

**Comment by Fourdee** [comment]
Notes:
Ok, waiting for https://api.github.com/repos/Radarr/Radarr/releases to be updated with 0.2.0.984> version. Used in our scrape to obtain the latest version available.

Once they do this (assuming the same sonarr fixed is applied), the issue of failing to run will no longer occur.

**Comment by Fourdee** [comment]
Radarr have updated their binary, resolves outstanding issues with latest mono library:
- 🈯️ Sonarr
- 🈯️ Radarr

Patched re-installs for v6.3, ETA release March 7th. Completed.
